### ISSUE TITLE
Sharing all features NOT limited but no edit/delete

### ISSUE THREAD

**Issue by passionate2023** [zadam/trilium#3405]
### Describe feature

The link https://github.com/zadam/trilium/wiki/Sharing says limited option

I have notebook with 2,000 subjects. I want to share the entire note book with all the features - search, highlight, advanced search etc., except edit/delete

Can you please add that feature or it is available already?

Thanks

**Comment by sigaloid** [comment]
This is not possible because it requires multi-user support. https://github.com/zadam/trilium/wiki/FAQ#multi-user-support

**Comment by zadam** [comment]
The sharing is a completely separate codebase from the main application - it's essentially a separate app (still living in the same process, though). The reasoning for that is that the "main trilium" app is large & complex and any bug would have the potential to leak data which should not have been published. By having only a simple/small sharing app, the risk of security issues is much reduced.

But as a result of that, the sharing feature set is drastically reduced. Anything needed has to be reimplemented. We can talk about adding specific features, but "all features NOT limited" is not going to happen.

**Comment by passionate2023** [comment]
There is no way to share all notebook via sharing option as there is no option in trilium....

However can you think of alternative or workaround to resolve this?

It would be easy if you can do the following....Example

a. when user creates a user id and password  => generate a user id and password for view => 2 credentials
b. send the credentials to the user
c. Disable the save and edit button based on the credentials

Simple words....
attach a role to the second credentials and you can solve the problem

Is it possible?

Basically instead of sharing...Giving a view credentials by disabling the menu save/update and delete

**Comment by zadam** [comment]
> Basically instead of sharing...Giving a view credentials by disabling the menu save/update and delete

No, that's not possible. Disabling menu items doesn't provide a lot of security, it's way more involved than that.

Supporting multiple users is an explicit non-goal:  https://github.com/zadam/trilium/wiki/FAQ#multi-user-support

**Comment by passionate2023** [comment]
Otherwise is it possible to synchronize note of another user to me....If that is simple, it can be used as a workaround...Basically my content needs to be shared with others without any hassles....whenever i did an update, delete or add, it should get reflected to the shared user or any user who can synchorize with my account

Basically pull the notes from my account to his account ....Uni-directional....

Similarly i can able to pull his notes with his user id and any code for transfer
